### PR TITLE
Simplify 01834_alias_columns_laziness_filimonov test

### PR DESCRIPTION
**Pull request by azat**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

**Comment by azat** [comment]
>[Fast test (actions) — fail: 1, passed: 3381, skipped: 655](https://s3.amazonaws.com/clickhouse-test-reports/36585/d8904673f40f0ec37e8f95226accb5c3d51a77ff/fast_test__actions_.html)

```
01268_procfs_metrics:                                                   [ FAIL ] 0.16 sec. - result differs with reference:·
--- /ClickHouse/tests/queries/0_stateless/01268_procfs_metrics.reference
+++ /ClickHouse/tests/queries/0_stateless/01268_procfs_metrics.stdout
@@ -1,2 +1 @@
-OSCPUVirtualTimeMicroseconds
 OSReadChars
```

This is interesting, sometimes it does not report counters, I also saw this in production, so I want to take a look, not just increase the time of this query.

**Comment by azat** [comment]
>This is interesting, sometimes it does not report counters, I also saw this in production, so I want to take a look, not just increase the time of this query.

- Will be fixed in #36602 - https://github.com/ClickHouse/ClickHouse/pull/36602/commits/35613e3010ca18e035f0cabf9cda7331352b284f

File: tests/queries/0_stateless/01834_alias_columns_laziness_filimonov.sh [hunk 1]
```diff
@@ -13,16 +13,20 @@ insert into aliases_lazyness(x) select * from numbers(40);
 # In very old ClickHouse versions alias column was calculated for every row.
 # If it works this way, the query will take at least 0.1 * 40 = 4 seconds.
 # If the issue does not exist, the query should take slightly more than 0.1 seconds.
-# The exact time is not guaranteed, so we check in a loop that at least once 
+# The exact time is not guaranteed, so we check in a loop that at least once
 # the query will process in less than one second, that proves that the behaviour is not like it was long time ago.
 
 i=0 retries=300
```
Comment (Algunenano) [inline tests/queries/0_stateless/01834_alias_columns_laziness_filimonov.sh:19]: Wouldn't it be possible to use sleep profile events instead of timing?

Comment (azat) [reply tests/queries/0_stateless/01834_alias_columns_laziness_filimonov.sh]: Indeed, good idea, thanks!

**Comment by azat** [comment]
>Conflicting files
>tests/queries/0_stateless/02271_fix_column_matcher_and_column_transformer.sql

- #36607 (fixed by @alesapin already)

Rebased.

**Comment by alexey-milovidov** [comment]
> Style Check (actions) — Check black failed. 

Fixed in master.

> Stress test (address, actions) — Backward compatibility check: Fatal message in clickhouse-server.log (see bc_check_fatal_messages.txt)

As expected (previous version had a bug)

**Comment by alexey-milovidov** [comment]
> 00170_s3_cache

This test needs attention @kssenii

**Comment by alexey-milovidov** [comment]
@mergifyio update

**Comment by kssenii** [comment]
> 00170_s3_cache
This test needs attention @kssenii

I think it is because of settings randomization which resulted in `max_threads=2` + it is `tsan` build. This test runs a lot of queries over hits dataset in s3 and already took significant time under tsan build. Having `max_threads=2` after settings randomization made it too slow and resulted in a timeout. I will disable settings randomization for this test.

**Comment by alexey-milovidov** [comment]
Integration test 

> test_ddl_worker_non_leader

needs attention from @tavplubix

**Pull request merged**